packages/web/types/CountdownCircleTimer.d.ts

```diff
@@ -25,6 +25,8 @@ export interface CountdownCircleTimerProps {
   size?: number
   /** Path stroke width. Default: 12 */
   strokeWidth?: number
+  /** Trail stroke width Default: false */
+  trailStrokeWidth?: number | boolean
```

Comment:
By the way, there is also file with type definitions for the mobile package, which will need to extend - https://github.com/vydimitrov/react-countdown-circle-timer/blob/master/packages/mobile/types/CountdownCircleTimer.d.ts